xcode/Subconscious/Shared/Models/Did.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Did.swift
+//  Subconscious
+//
+//  Created by Ben Follington on 1/3/2023.
+//
+
+import Foundation
+
+struct Did : Hashable, Identifiable {
```

Comment:
I couldn't find an explicit ruling in the Google Swift Styleguide RE: using acronyms in typenames. Should this be called `DID`? I've always camelcased these in the past but I see that in-built types like `URL` and `UI` opt for all caps.

Reply:
Apple tends to uppercase acronyms like URL. Otherwise camel case. I could go either way here.